resources/keywords/provision_dbinstance.resource

```diff
@@ -72,15 +72,24 @@ DBSC Instance Provisioned Successfully
     Click Button    ${btnViewDBInst}
 
 DBSC Instance Available On DataBase Access Page
-    Wait Until Keyword Succeeds     5x     2s   Wait Until Page Contains Element    ${titleDatabaseAccess}
+    Wait Until Keyword Succeeds     5x     2s   Filter DBSC Instance Until Success
+
+Filter DBSC Instance On Database Access Inventory Table
+    Wait Until Page Contains Element    ${titleDatabaseAccess}
     Element Should Be Visible    ${txtBxSearchByName}
     Input Text    ${txtBxSearchByName}    ${instanceName}
     ${elemInstanceNameUpd}=    Replace String    ${elemInstanceName}    <<inst-name>>    ${instanceName}
     ${elemInstProvNameUpd}=    Replace String    ${elemInstProvName}    <<inst-name>>    ${instanceName}
     ${elemInstProvNameUpd}=    Replace String    ${elemInstProvNameUpd}    <<paname>>    ${provaccname}
-    Element Should Be Visible    ${elemInstanceNameUpd}
+    Wait Until Element Is Visible    ${elemInstanceNameUpd}
```

Comment:
${elemInstanceNameUpd} should be changed to ${elemInstNameUpd}

Reply:
This Object has been reused from the existing declaration, IMHO we can work on our object naming convention after the guidelines and mapping has been defined. Please LMKWYT?